includes/Font.h

```diff
@@ -16,4 +15,4 @@ class Font
     virtual void drawText(const std::string& text, int x, int y) = 0;
 };
 
-} // namespace doengine
\ No newline at end of file
+}; // namespace doengine
```

Comment:
I think semicolon is not needed for namespaces
```suggestion
} // namespace doengine
```